src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -205,7 +205,7 @@
                     <v-list-item-group color="primary">
                       <v-list-item :disabled="disableCorrection(filing)">
                         <v-list-item-icon>
-                          <v-icon class="app-blue">mdi-file-document-edit-outline</v-icon>
```

Comment:
"app-blue" is the same as "primary".

Reply:
would it be worth it to get rid of app-blue just so we don't keep using it?

Reply:
No, we can't do that:
- Vuetify components accept `color` as "primary" (whatever is defined [here](https://github.com/bcgov/business-filings-ui/blob/3abbe109172f10baa031cc3504254d38d514edae/src/main.ts#L81))
- other components don't understand `color` and have to use a class instead to set the CSS color, ie, "app-blue"